tools/perf_checker/perf_checker.py

```diff
@@ -0,0 +1,42 @@
+import argparse
+
+import profiler
+
+# The script takes a command and a database path and looks
+# the performance anomalies in the performance history of that
+# command across the profiled runs.
+
+parser = argparse.ArgumentParser()
+parser.add_argument("command", type=str)
+parser.add_argument("db_path", type=str)
+
+args = parser.parse_args()
+
+# Processes the set of previously written logs
+# The threshold (ratio) of allowable performance degradation between profiling runs
+threshold = 1.10
+
+db = profiler.data.FileBasedProfileDB(args.db_path)
+dt = db.find(f"{args.command}")
+
+
+if len(dt) >= 2:
+    first_profile = dt[0]
+    curr_profile = dt[len(dt) - 1]
+    first_time = first_profile.elapsed_time_sec
+    curr_time = curr_profile.elapsed_time_sec
+
+    formatted_first_profile = str(first_profile).replace("\\n", "\n").replace("\\t", "\t")
+    formatted_curr_profile = str(curr_profile).replace("\\n", "\n").replace("\\t", "\t")
+
+    print(f"********************\n\nFirst profile:\n {formatted_first_profile}")
```

Comment:
I would recommend switching to using a logger and keep each log to a single line. Having multiple lines will make logs hard to read in an environment where lines are split. Consider using json logging if they span multiple lines. This may take a while so it can be done in a separate PR or just deferred to a later date.